src/relay/analysis/util.cc

```diff
@@ -450,6 +451,13 @@ bool IsDataDependant(const CallNode* call) {
         return false;
       }
     }
+  } else if (op->name == "topk") {
```

Comment:
We should also apply this to ones/zeros/full/broadcast_to.

Reply:
ones/zeros/full/broadcast_to don't need this since when shape is constant, they don't need shape func.